openqaoa/optimizers/result.py

```diff
@@ -34,6 +34,34 @@ def most_probable_bitstring(cost_hamiltonian, measurement_outcomes):
     return {'solutions_bitstrings': solutions_bitstrings,
             'bitstring_energy': bitstring_energy(cost_hamiltonian, solutions_bitstrings[0])}
 
+def lowest_cost_bitstring(cost_hamiltonian:Hamiltonian, measurement_outcomes:dict) -> dict:
```

Comment:
I think it would be nice if this would return your lowest-k energies, where `k` is an integer which defaults to 1 but in general is specified by the user

Reply:
done! Thanks for your suggestion.